releases/latest.releases.yaml

```diff
@@ -226,7 +226,7 @@ repositories:
   OpenXR:
     type: git
     url: https://github.com/KhronosGroup/OpenXR-SDK.git
-    version: release-1.0.22
+    version: release-1.0.23
```

Comment:
@S-Dafarra do you think this is ok or you prefer to stick to 1.0.22 ?

Reply:
I think there should be no problems in updating

Reply:
Ack!